### PR TITLE
Remove double write

### PR DESCRIPTION
**Pull request by mackosx**
## Description of the Change
Part 2 of https://app.shortcut.com/greenspace/story/77179/step-2-read-from-jti-instead-of-token-in-fork

Discussed below around do we need to be concerned about the unique -> null + blank migration?
https://app.shortcut.com/greenspace/story/77179/step-2-read-from-jti-instead-of-token-in-fork#activity-78426

## Checklist

- [x] Remove double write
- [x] Add migration
- [x] Increment version

### CODE REVIEW

**Comment by ryanluker** [comment]
@mackosx I added the thread for future reference.

File: oauth2_provider/models.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ class AbstractIDToken(models.Model):
         null=True,
         related_name="%(app_label)s_%(class)s",
     )
-    token = models.TextField(unique=True)
+    token = models.TextField(unique=True, null=True)
```
Comment (ryanluker) [inline oauth2_provider/models.py]: Do we want unique here anymore? I would have thought this would be null=True and blank=True

Comment (mackosx) [reply oauth2_provider/models.py]: `blank` just controls how it appears in the GSAdmin I think.. but I can update both!

Comment (ryanluker) [reply oauth2_provider/models.py]: TIL!
https://stackoverflow.com/a/21812150

**Review by ryanluker — APPROVED** [review]

**Comment by mackosx** [comment]
@ryanluker Going to merge this one so I can update the grnspace one. I think we are safe to merge that one to dev rightaway?

**Comment by ryanluker** [comment]
> @ryanluker Going to merge this one so I can update the grnspace one. I think we are safe to merge that one to dev rightaway?

Sure at this point I think that is safe enough!

**Pull request merged**